src/components/InstructorPage.js

```diff
@@ -0,0 +1,257 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router-dom'
+
+import Button from '@material-ui/core/Button'
+
+import './InstructorPage.css'
+
+//Services
+import peerReviewService from '../services/peerReview'
+
+//Actions
+//import instructorPageActions from '../reducers/actions/instructorPageActions'
+
+const GroupDetails = ({ myGroup }) => {
+  if (!myGroup) {
+    return (
+      <div>
+        <h2>This user is currently not part of any group.</h2>
+      </div>
+    )
+  } else {
+    return (
+      <div>
+        {myGroup.map((member, index) => {
+          return (
+            <p key={index}>
+              {index + 1}. {member}
+            </p>
+          )
+        })}
+      </div>
+    )
+  }
+}
+
+const Answers = ({ answers }) => {
+  if (answers) {
+    return (
+      <div>
+        {answers.map((projectGroup, index) => {
+          return (
+            <div key={index}>
+              <hr />
+              <br />
+              <h1>{projectGroup.group.name}</h1>
+              <h3>Ohjaaja: {projectGroup.group.instructorName}</h3>
+              <GroupDetails myGroup={projectGroup.group.studentNames} />
+
+              {projectGroup.round1Answers.length > 0 ? (
+                <div>
+                  <h2>Ensimmäisen vertaisarvioinnin vastaukset</h2>
+                  <GroupAnswers answers={projectGroup.round1Answers} />
+                </div>
+              ) : null}
+
+              {projectGroup.round2Answers.length > 0 ? (
+                <div>
+                  <h2>Toisen vertaisarvioinnin vastaukset</h2>
+                  <GroupAnswers answers={projectGroup.round2Answers} />
+                </div>
+              ) : null}
+
+              <br />
+            </div>
+          )
+        })}
+      </div>
+    )
+  } else {
+    return (
+      <div>
+        <p>Loading</p>
+      </div>
+    )
+  }
+}
+
+const getQuestions = (answers) => {
+  // The same questions are found in every answer, so just grab the answers
+  // from the first one
+  return answers[0].answer_sheet.map((question) => {
+    return {
+      type: question.type,
+      questionHeader: question.questionHeader
+    }
+  })
+}
+
+const Question = ({ title, children }) => (
+  <div>
+    <h3>{title}</h3>
+    {children}
+  </div>
+)
+
+const GroupAnswers = ({ answers }) => {
+  return (
+    <div>
+      {getQuestions(answers).map((question, index) => {
```

Comment:
Arrow function should always return something. No there is no else

Reply:
Added an empty else to prevent program from crashing in case there are no radio/text questions in instructor review.(Which should never happen anyway.)